src/main/java/net/imglib2/RandomAccessible.java

```diff
@@ -50,6 +50,14 @@
  * expect that the domain is infinite. (In contrast to this,
  * {@link RandomAccessibleInterval}s have a finite domain.)
  * </p>
+ * <p>
```

Comment:
I do not think this is true.  Typically, a RandomAccessible is expected to be defined everywhere.  Where is this not true (while not being an errror)?